prow/tekton.md

```diff
@@ -0,0 +1,142 @@
+# Using Prow with Tekton Pipeline
+
+The Prow agent type `tekton-pipeline` will allow you to trigger
+execution of
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) 
+with Prow!
+
+1. [Setting up Pipelines with Prow](#setting-up-pipelines-with-prow)
+2. [Configuring a `tekton-pipeline` job](#configuring-a-tekton-pipeline-job)
+
+**Warning**
+
+Prow users may expect to see logs from their `PipelineRuns` be visible in
+Gubernator, however this functionality is not supported yet, and logs must be
+grabbed directly from the running pods.
+
+## Setting up Pipelines with Prow
+
+1. [Setup and configure Prow itself](getting_started_deploy.md)
+2. [Add the `pipeline` and `crier` controllers](#pipeline-and-crier-controllers)
+3. [Install Tekton Pipelines](#install-tekton)
+
+### `pipeline` and `crier` controllers
+
+In addition to the services required by `Prow` for `prowjobs`, you will need
+to add:
+
+1. [The pipeline controller](/prow/cmd/pipeline): Creates
+   [`PipelineRuns](https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md)
+2. [The crier controller](/prow/cmd/crier): Updates GitHub with the results of
+   `ProwJobs` executed by agent types other than `kubernetes`
+
+You can add these to your cluster with:
+
+```bash
+# Add the service account for the crier controller
+k apply -f prow/cluster/crier_rbac.yaml
+
+# Add the service account for the pipeline controller
+k apply -f prow/cluster/pipeline_rbac.yaml
+
+# Add the pipeline controller deployment
+k apply -f prow/cluster/simple_pipeline_deployment.yaml
+
+# Add the crier controller deployment
+k apply -f prow/cluster/simple_crier_deployment.yaml
+
+```
+
+### Install Tekton Pipelines
```

Comment:
I am not sure but I think this step needs to be before installing the controller because the controller expects the PipelineRun CRD to be already installed in order to be able to start watching.

Reply:
kk i can try to make that more clear